pydsa/count_inversions.py

```diff
@@ -0,0 +1,45 @@
+	# Counting inversions in an array
+	# Complexity : O(nlog(n))
+
+
+def _merge(L, R):
```

Comment:
This function is already present here : https://github.com/pydsa/pydsa/blob/master/pydsa/merge_sort.py#L5